src/AppBundle/Controller/ComponentController.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+	namespace AppBundle\Controller;
+
+
+	use AppBundle\Entity\Component;
+	use AppBundle\Form\ComponentType;
+	use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+	use Symfony\Component\HttpFoundation\Request;
+	use Ob\HighchartsBundle\Highcharts\Highchart;
+
+
+	class ComponentController extends Controller
+	{
+		/***
+		* @param $dashboardId
+		* @return \Symfony\Component\HttpFoundation\RedirectResponse
+		*/
+		public function addComponentAction($dashboardId)
```

Comment:
You could use here ParamConverter http://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/annotations/converters.html